asf_public_discourse_home_decarbonisation/analysis/buildhub/sentence_topic_and_sentiment_analysis_for_bh.py

```diff
@@ -0,0 +1,332 @@
+            "129_immersion_immersions_3kw_solic",
+        ],
+        "Pumps and Circulators": [
+            "15_pump_pumps_speed_circulation",
+            "147_ftc6_ftc_ftc5_controller",
+        ],
+        "Thermostats and Controllers": [
+            "36_thermostat_thermostats_room_control",
+            "55_controller_controls_control_controllers",
+            "91_settings_setting_parameters_change",
+            "141_ecodan_controller_ftc6_cooling",
+        ],
+        "Radiators and Manifolds": [
+            "19_radiators_radiator_output_temperature",
+            "34_manifold_manifolds_mixer_blending",
+            "40_rads_rad_warm_temp",
+        ],
+        "Compressors and Refrigerants": [
+            "43_compressor_compressors_scroll_crankcase",
+            "89_refrigerant_fridge_refrigeration_refrigerants",
+            "133_glycol_ethylene_exchanger_propylene",
+        ],
+        "Fans and Coils": [
+            "77_floor_tiles_flooring_floors",
+            "86_inverter_driven_inverters_drive",
+        ],
+        "Noise": [
+            "4_noise_sound_noisy_quiet",
+        ],
+        "Miscellaneous Components": [
+            "49_flow_rate_rates_meter",
+            "50_wiring_cable_wires_wire",
+            "65_zones_zone_zoning_single",
+            "121_phase_single_3phase_transformer",
+            "127_blinds_external_windows_velux",
+            "135_warranty_year_manufacturer_item",
+            "137_trvs_trv_open_rads",
+            "143_batteries_battery_storage_life",
+        ],
+    },
+    "Installation": [
+        "10_installer_installers_install_installation",
+        "25_plumber_plumbers_plumbing_electrician",
+        "94_vaillant_daikin_installer_schematics",
+        "118_manual_manuals_instructions_copy",
+    ],
+    "Temperature and Control": [
+        "13_flow_temp_temperature_temps",
+        "36_thermostat_thermostats_room_control",
+        "55_controller_controls_control_controllers",
+        "79_degrees_temperature_temp_set",
+        "91_settings_setting_parameters_change",
+        "147_ftc6_ftc_ftc5_controller",
+    ],
+    "Certification and Incentives": [
+        "21_certification_microgeneration_scheme_certificate",
+        "51_grant_bus_grants_5k",
+        "96_epc_incentive_renewable_rating",
+        "136_certification_microgeneration_scheme_certified",
+    ],
+    "Energy and Efficiency": [
+        "52_carbon_co2_emissions_fossil",
+        "62_octopus_tariff_cosy_agile",
+        "115_efficiency_efficient_inefficient_penalty",
+        "108_tariff_tariffs_economy_tou",
+        "126_energy_consumption_efficiency_rating",
+    ],
+    "Design and Planning": [
+        "67_planning_lpa_pd_permission",
+        "69_size_sizing_oversizing_fit",
+        "111_design_designers_designer_professional",
+    ],
+    "Weather and Climate": [
+        "32_winter_weather_cold_rain",
+        "85_house_warm_cold_days",
+        "110_climate_change_global_scientists",
```

Comment:
I think Climate change should potentially go in **Other**/**Miscellaneous** group. This is likely to be very different from chats about cold weather and cold days, when thinking about HPs.